tsconfig.json

```diff
@@ -22,7 +22,7 @@
     "skipLibCheck": true
   },
   "vueCompilerOptions": {
-    "experimentalAllowTypeNarrowingInInlineHandlers": true
+    "narrowingTypesInInlineHandlers": true
```

Comment:
https://github.com/johnsoncodehk/volar/blob/master/CHANGELOG.md#100-rc0-2022105

renamed in Volar